src/admin/sections/TokenGen.module.scss

```diff
@@ -0,0 +1,34 @@
+.tokenNumberInput,
+.button {
+  border: none;
+  background-color: var(--input-background);
+  color: var(--input-color);
+  padding: 5px 10px;
+  border-radius: 3px;
+  font-family: inherit;
+  font-size: 1rem;
+  text-decoration: none;
```

Comment:
You are using this exact class so often in multiple different stylesheets. Think you should make a component for it.
And maybe add `cursor: pointer` to it, the interaction seems so bland atm.